draft-sharabayko-srt-over-quic.md

```diff
@@ -75,22 +75,22 @@ informative:
 
 --- abstract
 
-This document presents an approach to tunnel SRT live streaming over QUIC datagrams.
+This document presents an approach to tunnelling SRT live streams over QUIC datagrams.
 
 QUIC {{RFC9000}} is a UDP-based transport protocol providing TLS encryption, stream multiplexing,
-connection migration. It was designed to become a faster alternative to the TCP protocol {{RFC7323}}.
+and connection migration. It was designed to become a faster alternative to the TCP protocol {{RFC7323}}.
 
 An Unreliable Datagram Extension to QUIC {{QUIC-DATAGRAM}} adds support for sending and receiving
 unreliable datagrams over a QUIC connection.
 
-SRT {{SRTRFC}} is a UDP-based transport protocol. Essentially it can operate over any unreliable datagram transport.
-SRT in live streaming configuration provides end-to-end latency-aware mechanisms for packet loss recovery.
-If SRT fails to recover a packet loss within the specified latency, then the packet is lost finally (dropped) to avoid
+SRT {{SRTRFC}} is a UDP-based transport protocol. Essentially, it can operate over any unreliable datagram transport.
+SRT in live streaming configuration provides an end-to-end latency-aware mechanisms for packet loss recovery.
+If SRT fails to recover a packet lost within a specified latency, then the packet is dropped to avoid
```

Comment:
```suggestion
If SRT fails to recover a packet loss within a specified latency, then the packet is dropped to avoid
```

Recover packet loss until the latency restriction permits. Not a packet lost within the latency.